### PR TITLE
Driver

### PR DESCRIPTION
**Pull request by Fankhauser-Dominik**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--eds-self-service-website--headwire-edge-delivery.hlx.live/
- After: https://driver--eds-self-service-website--headwire-edge-delivery.hlx.live/

### CODE REVIEW

File: blocks/site-details/site-details.js [hunk 1]
```diff
@@ -1272,12 +1266,13 @@ export default async function decorate(block) {
...
         })
         .catch((error) => {
+          // eslint-disable-next-line no-console
           console.log(error);
         });
 
       // MARK: add page button
       block.querySelector('.add-page').onclick = () => {
-        addPageDialogSetup({ project, headers, darkAlleyVariation });
+        addPageDialogSetup({ project, headers });
```
Comment (icaraps) [inline blocks/site-details/site-details.js:1275]: Why did you remove `darkAlleyVariation` ?

File: blocks/wizard/wizard.js [hunk 2]
```diff
@@ -177,18 +177,7 @@ export default async function decorate(block) {
           <a href="/templates/${id}" id="${id}" class="template ${i === 0 ? 'is-selected' : ''}">
             <h3>${name}</h3>
             <p>${description}</p>
-            <div class="carousel">
```
Comment (icaraps) [inline blocks/wizard/wizard.js:180]: Why did you remove the carousel ? Is it because you're not supposed to allow other clickable elements inside a `<a>` ?

File: styles/styles.css [hunk 3]
```diff
@@ -534,3 +534,106 @@ img[src="/icons/loading.svg"] {
...
+}
+
+
+#expedition-popover-title, #expedition-popover-footer {
+  background-color: transparent;
+  color: black;
+  height: auto;
+}
+
+.expedition-popover-btn, .expedition-popover-prev-btn, .expedition-popover-next-btn, .disable-tour-btn {
```
Comment (icaraps) [inline styles/styles.css:624]: Would it be possible for consistency to reuse the button styles we already use across kestrelone ?

File: package.json [hunk 4]
```diff
@@ -19,5 +19,8 @@
     "eslint-plugin-import": "2.29.1",
     "stylelint": "16.3.1",
     "stylelint-config-standard": "36.0.0"
+  },
+  "dependencies": {
+    "driver.js": "^1.3.1"
```
Comment (icaraps) [inline package.json:24]: Sorry I don't see where driver.js is being imported ?

**Comment by icaraps** [comment]
There's another PR https://github.com/headwire-edge-delivery/eds-self-service-website/pull/24 which we could merge before but then it means we would have to add it to the tour.

File: package.json (hunk 4 above)
Comment (Fankhauser-Dominik) [reply package.json:24]: Thanks, that's unused.

File: blocks/wizard/wizard.js (hunk 2 above)
Comment (Fankhauser-Dominik) [reply blocks/wizard/wizard.js:180]: We decided to replace the carousel with a static image.
It was a little bit hard to change between the images.
A Single Image for a quick preview should be okay, as Users can preview the Site.

File: blocks/site-details/site-details.js (hunk 1 above)
Comment (Fankhauser-Dominik) [reply blocks/site-details/site-details.js:1275]: #21 should have added it to my PR, but somehow it wasn't applied.

**Comment by icaraps** [comment]
@DriftingSands can you pls review as well since the PR affects everything its better to have more eyes on it.

**Review by icaraps — APPROVED** [review]

**Comment by DriftingSands** [comment]
image
I guess before, all cards just happened to be the same size. We should use `flex: 1;` now

**Comment by DriftingSands** [comment]
image
Tour did not automatically start here & I think I got the wrong one after clicking help button.

We should tell users what each button here does, and guide them through. And it should open automatically

**Comment by icaraps** [comment]
> I guess before, all cards just happened to be the same size. We should use flex: 1; now

We should retake the screenshots for all templates and make sure they have all same dimensions. That can be done later.

**Comment by DriftingSands** [comment]
> > I guess before, all cards just happened to be the same size. We should use flex: 1; now
> 
> We should retake the screenshots for all templates and make sure they have all same dimensions. That can be done later.

Even then I think flex: 1 as a fallback makes sense

**Comment by DriftingSands** [comment]
image
this pops up at the end of every tour. It's quite annoying, as I don't want to disable the automatic tour yet. I haven't seen everything.

Also the tour doens't trigger automatically anymore anywhere? I haven't been on site-details before, I keep having to click help to get it to show up

**Comment by DriftingSands** [comment]
Update section should get a small tour too

**Comment by Fankhauser-Dominik** [comment]
> image I guess before, all cards just happened to be the same size. We should use `flex: 1;` now

image

It wasn't the same size before.
It's becaus of this:
`.wizard.block .template {
    margin: auto auto 32px;
    }`

**Comment by DriftingSands** [comment]
Next, Prev, Done button on the tour pop-ups have no visible keyboard focus. Theres a small animation, but its hard to spot, start & end is the same style

**Comment by icaraps** [comment]
> We should retake the screenshots for all templates and make sure they have all same dimensions. That can be done later.

I just did it. Not sure if its automatically reflected on this branch ...
